src/scenes/Event/scenes/topic.js

```diff
@@ -471,9 +466,9 @@ export default class TopicPage extends React.Component {
     return (
       <div className={classes.withdrawOptionsWrapper}>
         {_.map(topic.options, (option, index) => (
-          <div key={`option-${index}`} className={classNames(classes.withdrawContainerSection, 'option')}>
+          <div key={`option-${index}`} className={cx(classes.withdrawContainerSection, 'option')}>
             <div className={classes.eventOptionNum}>{index + 1}</div>
-            <Typography variant="title" className={topic.resultIdx === index ? classes.withdrawWinningOptionSmall : null}>
+            <Typography variant="title" className={cx({ [classes.withdrawWinningOptionSmall]: topic.resultIdx === index })}>
```

Comment:
do we need `cx` here since its just one item?

Reply:
we could do it like:
A. `cx({ [classes.withdrawWinningOptionSmall]: topic.resultIdx === index })`
B. `topic.resultIdx === index && classes.withdrawWinningOptionSmall`
Now that I think about it, I guess B is probably more readable :)